### DISCUSSION TITLE
Add Wifi current channel used in Information screen.

### DISCUSSION THREAD

**Discussion by OfferLa** [arendst/Tasmota#18153] [Ideas]
Just a little, simple thing.

In the cli, there is an option to see the current channel that is used for the WiFi communication with the AP.

In the info screen (web interface) it just shows the AP name protocol type (e.g. 11n) and dBm.

In my house - I have several APs (same SSID and password) to make good availability. It would be really helpful to know (according to the channel) what AP is used right now.

Many thanks for a great work!!!

**Comment by Jason2866** [comment]
@arendst  Would be a usefull info. Maybe just right adding after SSID before the RSSI?

**Reply by barbudor** [reply]
Channel is not accurate to know which AP you are connected to as nothing prevent you to have all your APs on the same channel (I don't known which is more efficient but I also chose to have different channels on each AP)

If this is the exact goal, it would be better to consider the BSSId. This is what I'm checking (in TDM) when I want to check to which AP my devices are connected.

**Reply by sfromis** [reply]
Right, I've got a backend tool where I transform the BSSID to a short readable name, and the channel number is only a details of interest. Even more interesting if the AP selected a channel number which I do not "agree" with.

**Comment by arendst** [comment]
Added just now 

![image](https://user-images.githubusercontent.com/11044339/224495700-07260294-3759-45f1-a0e9-1303f8d951ba.png)

although I find the info on one line becoming a mess but hey.....

**Reply by OfferLa** [reply]
Dear @arendst ,
You are my hero!
Thank you so much for this! (and for your all of your great work in general!)
All the best!!!
